plugin/adsk/scripts/CMakeLists.txt

```diff
@@ -10,6 +10,7 @@ list(APPEND scripts_src
     mayaUsd_createStageFromFile.mel
     mayaUsd_createStageWithNewLayer.py
     mayaUsd_createStageFromAsset.mel
+	mayaUsd_help.py
```

Comment:
it seems that this is a tab vs spaces issue. I'll update this file to match the other lines.